lib/matplotlib/quiver.py

```diff
@@ -520,11 +532,9 @@ def get_datalim(self, transData):
     @allow_rasterization
     def draw(self, renderer):
         self._init()
-        if (self._new_UV or self.angles == 'xy'
-                or self.scale_units in ['x', 'y', 'xy']):
-            verts = self._make_verts(self.U, self.V)
-            self.set_verts(verts, closed=False)
-            self._new_UV = False
+        verts = self._make_verts(self.U, self.V)
```

Comment:
Would you mind just commenting (here, not in the code) what the change represents. It seems that before scale_units was used, and now it isn't.